disclosure_backend/settings.py

```diff
@@ -1,7 +1,8 @@
 import os
+import os.path as op
```

Comment:
Yep, I am primarily a nodejs developer, so I'm fine if you want to push back based on the python/django conventions.